generic_relations/tests/test_relations.py

```diff
@@ -57,6 +58,9 @@ def setUp(self):
         self.note = Note.objects.create(text='Remember the milk')
         Tag.objects.create(tagged_item=self.note, tag='reminder')
 
+        Detachable.objects.create(content_object=self.note, name='attached')
+        Detachable.objects.create(content_object=None, name='detached')
```

Comment:
An object cannot be created using `create` with the `GenericForeignKey` being a null value. [You end up triggering an exception](https://github.com/django/django/blob/f61c4f490dc4c8ec6ba94ad4f40247db2425fc3e/django/contrib/contenttypes/fields.py#L177-L178).